src/platform/site-wide/sass/merger.scss

```diff
@@ -913,6 +913,106 @@
       z-index: 2;
     }
   }
+
+  .va-sidebarnav {
+    h4 {
```

Comment:
It sounds like this will be removed later on, since the styles were copied and pasted for the time being. We should add a comment explaining that.

Reply:
Yes this will get removed once we update formation. I'll make sure to cross reference both to make sure we don't have repeat code.